features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -803,7 +806,9 @@
   "myaccount.overview.enabled": true,
   "myaccount.personal_info.enabled": true,
   "myaccount.security.enabled": true,
-
+  "myaccount.product_version.configs": {
+    "versionOverride": "BETA"
+  },
```

Comment:
myaccount will have the product version... as it is released with 5.10.0

Reply:
Fixed with `5125200`